### PR TITLE
Fixed line chart events test case

### PR DESCRIPTION
**Pull request by v-asurse**
Fixed line chart events test case which helped in increasing EventAnnotations code coverage. 
LineChartEvents

### CODE REVIEW

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx [hunk 1]
```diff
@@ -627,13 +627,16 @@ describe.skip('Line chart - Subcomponent Event', () => {
...
     LineChart,
-    { data: simplePoints, eventAnnotationProps, tickValues, tickFormat: '%m/%d' },
+    { data: simplePoints, eventAnnotationProps: eventAnnotationProps },
     container => {
       // Arrange
-      const event = screen.queryByText('3 events');
-      // Assert
-      expect(event).toBeDefined();
-      fireEvent.click(event!);
+      const svg = container.querySelector('svg');
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx:633]: dont use queryselector with RTL. refer RTL documentation on how to get elements to validate in RTL

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx [hunk 2]
```diff
@@ -627,13 +627,16 @@ describe.skip('Line chart - Subcomponent Event', () => {
   testWithWait(
     'Should render events with defined data',
     LineChart,
-    { data: simplePoints, eventAnnotationProps, tickValues, tickFormat: '%m/%d' },
+    { data: simplePoints, eventAnnotationProps: eventAnnotationProps },
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx:630]: what was the issue. How is the test case passing now

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx (hunk 2 above)
Comment (v-asurse) [reply packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx:630]: queryByText is not returning the events data so we have updated it to getElementsByTagName also removed unwanted props.

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx (hunk 1 above)
Comment (v-asurse) [reply packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx]: We have updated the querySelector with getElementsByTagName.

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx (hunk 2 above)
Comment (AtishayMsft) [reply packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx:630]: get by tag should not be used for RTL.
See the suggested methods to use https://testing-library.com/docs/react-testing-library/cheatsheet

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx [hunk 3]
```diff
@@ -614,7 +614,7 @@ describe('Line chart - Subcomponent xAxis Labels', () => {
   );
```
Comment (fabricteam) [inline packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx:614]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31618&commitId=d709e5b4c13462008297a38f1d7e0640451f9a29&env=prod&clientType=FLUENTUI)


  <details><summary><b>Pivot - Overflow</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx [hunk 4]
```diff
@@ -627,13 +627,22 @@ describe.skip('Line chart - Subcomponent Event', () => {
   testWithWait(
     'Should render events with defined data',
     LineChart,
-    { data: simplePoints, eventAnnotationProps, tickValues, tickFormat: '%m/%d' },
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/LineChart/LineChartRTL.test.tsx:630]: why removing tick format

**Review by AtishayMsft — APPROVED** [review]

**Comment by AtishayMsft** [comment]
@v-asurse Could you respond to the Contributor License Agreement comment.

**Comment by v-asurse** [comment]
> @v-asurse please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

**Comment by v-asurse** [comment]
> > @v-asurse please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> > ```
> > @microsoft-github-policy-service agree [company="{your company}"]
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > > Options:
> > > 
> > > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > > 
> > > ```
> > > @microsoft-github-policy-service agree
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > 
> > > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > > 
> > > ```
> > > @microsoft-github-policy-service agree company="Microsoft"
> > > ```
> > 
> > 
> > Contributor License Agreement

**Comment by v-asurse** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Pull request merged**